telegraf.go

```diff
@@ -39,9 +42,18 @@ type ClientImpl struct {
 //
 // > client := NewClientImpl("tcp://127.0.0.1:8094")
 func NewClientImpl(addr string) (*ClientImpl, error) {
-	// }
 	conn, err := createDialConn(addr)
-	return &ClientImpl{conn}, err
+	defaultTags := new(map[string]interface{})
```

Comment:
Not sure if this is the proper way to set this up

Reply:
Ive typically use `make` instead of `new` since it initializes the object. Pretty sure new just returns a pointer without initing.